kafka/consumer.go

```diff
@@ -228,6 +231,7 @@ func (c *Consumer) fetch(ctx context.Context) error {
 			c.cfg.Logger.Error("unable to process event",
 				zap.Error(err),
 				zap.String("topic", msg.Topic),
+				zap.ByteString("message.value", msg.Value),
 				zap.Int64("offset", msg.Offset),
 				zap.Int32("partition", msg.Partition),
 				zap.Any("headers", meta),
```

Comment:
**question**: should we use a more general name for the log field ? `metadata` maybe ?

Reply:
ideally both Kafka and pubsublite should have the same field. Either headers or metadata seem fine to me.

Reply:
Let's keep headers then!